holoviews/plotting/bokeh/util.py

```diff
@@ -434,17 +468,24 @@ def make_axis(axis, size, factors, dim, flip=False, rotation=0,
         height = int(axis_height + np.abs(np.sin(rotation)) *
                      ((nchars*tick_px)*0.82)) + tick_px + label_px
         opts = dict(x_axis_type='auto', x_axis_label=axis_label,
-                    x_range=ranges, y_range=ranges2, plot_height=height,
-                    plot_width=size)
+                    x_range=ranges, y_range=ranges2)
+        if bokeh3:
+            opts = {**opts, 'height': height, 'width': size}
+        else:
+            opts = {**opts, 'plot_height': height, 'plot_width': size}
     else:
         # Adjust width to compensate for label rotation
         align = 'left' if flip else 'right'
         width = int(axis_height + np.abs(np.cos(rotation)) *
                     ((nchars*tick_px)*0.82)) + tick_px + label_px
         opts = dict(y_axis_label=axis_label, x_range=ranges2,
-                    y_range=ranges, plot_width=width, plot_height=size)
+                    y_range=ranges)
+        if bokeh3:
+            opts = {**opts, 'height': size, 'width': width}
+        else:
+            opts = {**opts, 'plot_height': size, 'plot_width': width}
```

Comment:
Lots of height/plot_height and width/plot_width handling here, might have suggested just defining `HEIGHT_NAME` and `WIDTH_NAME` globals instead of all the conditional handling here but not a priority.

Reply:
That was definitely also a possibility. I might have done it that way if I had started making it work with Bokeh 2 and Bokeh 3, not only Bokeh 3.

The reason I say might is that the code then becomes a lot of `setattr` and `getattr`, which makes the code less clear than having if statements. But that may be a personal opinion.

Reply:
> `opts = {**opts, 'height': size, 'width': width}`

This should work as far back as bokeh 2.3. Does holoviews need even older bokeh?

Reply:
No, because of the pin of panel 0.13.1, we only support bokeh 2.4.0 and up. 

I haven't seen any deprecation warnings for `plot_height` and `plot_width`, so I assumed it was a change made in Bokeh 3.0. 

I will fix this.